packages/misc/src/lib/misc.ts

```diff
@@ -27,10 +27,10 @@ import {
 } from '@lit-protocol/types';
 import { JsonRpcProvider } from '@ethersproject/providers';
 import { Contract } from '@ethersproject/contracts';
-import { LogLevel, LogManager } from '@lit-protocol/logger';
+import { LogManager } from '@lit-protocol/logger';
 import { version } from '@lit-protocol/constants';
 
-const logBuffer: Array<Array<any>> = [];
+const defaultLogger = LogManager.Instance.get('lit-sdk');
```

Comment:
will this be a global logger? I would like to keep the package names as categories if possible. Open to discussion however.